### ISSUE TITLE
[Quest] The Prophecy of Akida

### ISSUE THREAD

**Issue by Sinyr** [Atlantiss/NetherwingBugtracker#170]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
**Current behaviour**:
Freeing 1 Stillpine Captive counts as if you rescued 2. It doesn't look as if there were 2 NPCs stacked in each other either.
**Expected behaviour**: 
Freeing a Captive should progress the quest by 1.
**Server Revision**: 
2028

**Comment by Kiperr** [comment]
Did u clean cache befoure testing?

**Comment by Sinyr** [comment]
Yes, I cleared the cache before I started playing on this server.

**Comment by Peglaa** [comment]
Worked fine for me as well yesterday when I did it.
